Add TikTok icon in social buttons (#1778)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/content/docs/5.3/components/footer.md

### Follow us bar

This footer bar is a container with `.footer-social` class into which a list of social media icons can be added.

{{< callout info >}}

> Other social icons are available. See full list in [Social buttons section]({{< docsref "/components/buttons#social-buttons" >}}).

Reviewer: ```suggestion
See the full list of our provided socials in our [Social buttons section]({{< docsref "/components/buttons#social-buttons" >}}).
```
